services-directions/src/test/java/com/mapbox/api/directions/v5/models/RouteOptionsTest.java

```diff
@@ -214,4 +215,94 @@ public void toJson_fromJson() throws Exception {
 
     assertEquals(routeOptions, routeOptionsFromJson);
   }
+
+  @Test
+  public void test_fromJson() {
+    String jsonString = "{" +
+      "\"profile\": \"auto\"," +
+      "\"user\": \"mapbox\"," +
+      "\"baseUrl\": \"https://api.mapbox.com\"," +
+      "\"coordinates\": [[-3.707788,40.395039],[-3.712179,40.401819]]," +
+      "\"access_token\": \"ACCESS_TOKEN\"," +
+      "\"geometries\": \"polyline6\"," +
+      "\"overview\": \"full\"," +
+      "\"steps\": true," +
+      "\"bearings\": \";\"," +
+      "\"continue_straight\": true," +
+      "\"annotations\": \"congestion,distance\"," +
+      "\"language\": \"en\"," +
+      "\"roundabout_exits\": true," +
+      "\"voice_instructions\": true," +
+      "\"banner_instructions\": true," +
+      "\"voice_units\": \"imperial\"," +
+      "\"uuid\": \"uuid1\"" +
+      "}";
+
+    RouteOptions routeOptions = RouteOptions.fromJson(jsonString);
+    assertEquals("auto", routeOptions.profile());
```

Comment:
Let's add a new line here to remark the [triple A](https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40) "rule".